Update how-to-prepare-for-teacher-training.md (#87)
DFE-Digital/teacher-success: app/views/content/prepare-for-training/how-to-prepare-for-teacher-training.md

You can also try the <%= govuk_link_to "realistic job preview tool", "https://platform.teachersuccess.co.uk/p/XK4rV0xN16/41xZQ10Z2l?id=c74fef2451c5af60c516075028d313d55321e06d2d53232082e329e292611d5b" %> where you can respond to realistic video scenarios to understand what life in the classroom is like.

> You do not have to have classroom experience before you start training, but it could help you feel better prepared.

Reviewer: ```suggestion
You don't have to have classroom experience before you start training, but it could help you feel better prepared.
```
